cmd/gpu-kubelet-plugin/cdi.go

```diff
@@ -139,7 +142,79 @@ func NewCDIHandler(opts ...cdiOption) (*CDIHandler, error) {
 	return h, nil
 }
 
+func (cdi *CDIHandler) writeSpec(spec spec.Interface, specName string) error {
```

Comment:
Can we mvoe this to the bottom of the file. We generally prefer to have private methods below public ones.